src/uct/cuda/cuda_ipc/cuda_ipc_iface.c

```diff
@@ -276,6 +283,8 @@ static ucs_status_t uct_cuda_ipc_iface_query(uct_iface_h tl_iface,
     iface_attr->cap.get.align_mtu       = iface_attr->cap.get.opt_zcopy_align;
     iface_attr->cap.get.max_iov         = 1;
 
+    /* latency and overhead aren't depend on config values for wire
+     * compatibility sake */
```

Comment:
Latency and overhead don't depend on config values for the sake of wire compatibility.

Reply:
unresolved